package.json

```diff
@@ -86,6 +86,7 @@
     "hardhat": "^2.13.0",
     "hardhat-contract-sizer": "^2.8.0",
     "hardhat-gas-reporter": "^1.0.9",
+    "mocha": "^10.2.0",
```

Comment:
Is this needed?

Reply:
It's not needed since it's a dependency of (and used by) hardhat, but we do use it to run our e2e tests, so adding it here might make sense.

Reply:
cool. @anudit can you `npm i` to update package lock? looks like the linked version from hardhat is one patch version older.

Reply:
@joewagner Yeah, ran into an error because `mocha` wasn't installed while running tests. 
@sanderpick Done.